manticore/platforms/evm.py

```diff
@@ -1385,12 +1509,18 @@ def POP(self, a):
         # by the instruction dispatcher
         pass
 
+    def MLOAD_gas(self, address):
+        return self._get_memfee(address + 32)
+
     def MLOAD(self, address):
         '''Load word from memory'''
         self._allocate(address, 32)
         value = self._load(address, 32)
         return value
 
+    def MSTORE_gas(self, address, value):
+        return self._get_memfee(address + 32)
```

Comment:
It's probably good to add a comment why is it +32 or +1 here and there (at different funcs)

Reply:
Fixed/improved  in _get_memfee (and _use_data)